src/main/java/com/google/sps/data/AQDataPoint.java

```diff
@@ -4,10 +4,15 @@
  * This class represents the unified data format for AQ data passed to the client.
  */
 public class AQDataPoint {
-    private int lat;
-    private int lng;
-    private double aqi;
-    private String siteName;
+  public double lat;
+  public double lng;
+  public double aqi;
+  public String siteName;
 
-    public AQDataPoint() {}
+  public AQDataPoint() {}
```

Comment:
Empty default constructor should be omitted.

Reply:
yes, and i think to add constructor with args is recommended, as the canonical way to instantiate a class is `new AQDataPoint(lat, lgn, aqi, siteName)`. and it is simpler compared to the approach of updating fields one by one after creating a default instance of class.